### PR TITLE
modification on some docs

### PR DESCRIPTION
**Pull request by bjjwwang**
1. sync index page Chinese README with English README
2. check every hyperlink and remove dead links.
3. Change some literature using according to QA's recommendations.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ curl -H "Content-Type:application/json" -X POST -d '{"words": "我爱北京天
 ```
 - **Request result**: 
 ``` shell
-{"word_seg":"我|爱|北京|天安门"}
+{"word_seg":"我|爱|北京|天安门" or {"word_seg":"\u6211|\u7231|\u5317\u4eac|\u5929\u5b89\u95e8"} if Chinese Dataset is not installed}
```
Comment (guru4elephant) [inline README.md]: Chinese Dataset?

File: README.md [hunk 2]
```diff
@@ -162,26 +162,26 @@ curl -H "Content-Type:application/json" -X POST -d '{"url": "https://paddle-serv
 
 ### New to Paddle Serving
 - [How to save a servable model?](doc/SAVE.md)
-- [An end-to-end tutorial from training to serving(Chinese)](doc/TRAIN_TO_SERVICE.md)
-- [Write Bert-as-Service in 10 minutes(Chinese)](doc/BERT_10_MINS.md)
+- [End-to-end process from training to deployment](doc/TRAIN_TO_SERVICE.md)
```
Comment (guru4elephant) [inline README.md]: 这个标题，end-to-end process不是很容易懂吧，end-to-end tutorial应该没问题。另外，training to deployment->training to inference service deployment

File: README.md [hunk 3]
```diff
@@ -162,26 +162,26 @@ curl -H "Content-Type:application/json" -X POST -d '{"url": "https://paddle-serv
...
 - [How to config Serving native operators on server side?](doc/SERVER_DAG.md)
 - [How to develop a new Serving operator](doc/NEW_OPERATOR.md)
 - [Golang client](doc/IMDB_GO_CLIENT.md)
-- [Compile from source code(Chinese)](doc/COMPILE.md)
+- [Compile from source code](doc/COMPILE.md)
 
 ### About Efficiency
-- [How profile serving efficiency?(Chinese)](https://github.com/PaddlePaddle/Serving/tree/develop/python/examples/util)
-- [Benchmarks](doc/BENCHMARK.md)
+- [How to profile Paddle Serving efficiency?(Chinese)](https://github.com/PaddlePaddle/Serving/tree/develop/python/examples/util)
```
Comment (guru4elephant) [inline README.md]: efficiency -> latency?

**Review by guru4elephant — COMMENTED** [review]
please refine some words.

File: README.md (hunk 3 above)
Comment (bjjwwang) [reply README.md]: agree. I will fix it.

File: README.md (hunk 1 above)
Comment (bjjwwang) [reply README.md]: QA 说输出是 {"word_seg":"\u6211|\u7231|\u5317\u4eac|\u5929\u5b89\u95e8"} . 是没装中文字符集的问题。
要不我把这个移除了。

File: README.md (hunk 2 above)
Comment (bjjwwang) [reply README.md]: 好的好的，我改下

**Review by guru4elephant — APPROVED** [review]
LGTM

**Pull request merged**